src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -32,7 +33,6 @@ import MatrixClientPeg from '../../../MatrixClientPeg';
  * stageParams:            params from the server for the stage being attempted
  * errorText:              error message from a previous attempt to authenticate
  * submitAuthDict:         a function which will be called with the new auth dict
- * setSubmitButtonEnabled: a function which will enable/disable the 'submit' button
```

Comment:
github won't let me comment on it, but could you update the comment on line 24 to refer to InteractiveAuth rather than InteractiveAuthDialog?